quodlibet/quodlibet/ext/events/thumbrating.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# View Lyrics: a Quod Libet plugin for viewing lyrics.
+# Copyright (C) 2018 Eoin O'Neill (eoinoneill1991@gmail.com)
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+from gi.repository import Gtk
+from quodlibet import _
+from quodlibet.plugins.events import EventPlugin
+from quodlibet.qltk import Icons, ToggleButton
+from quodlibet.plugins.gui import UserInterfacePlugin
+
+
+class RatingBox(Gtk.VBox):
+    def __init__(self):
+        super(RatingBox, self).__init__(self)
+
+        self.thumb_ups = 1
+        self.thumb_downs = 1
+
+        self.title = Gtk.Label("")
+        self.title.set_line_wrap(True)
+        self.title.set_lines(2)
+
+        hbox = Gtk.HBox()
+        self.upvote = ToggleButton("👍", "")
+        self.downvote = ToggleButton("👎", "")
```

Comment:
I'm not entirely sure how cross-platform it is to use emojis as icons, but I suppose it should be fine. @lazka Do you have any input on this?

Reply:
> @lazka Do you have any input on this?

It might not look as nice everywhere (there is no color on Windows for example), but it should be fine.

Reply:
Ideally, this would use SVG but I found that emoji were a decent work around for getting the prototype ready and legible. I'm not sure on what the policy is with plugins that may require additional resources (like images or sounds.)